test/unit/modules/util/next-tick.spec.js

```diff
@@ -0,0 +1,25 @@
+import { nextTick } from 'core/util/env'
+
+describe('nextTick', () => {
+  it('accepts a callback', done => {
+    nextTick(done)
```

Comment:
I'd rather not force people to learn another method name. This convention of being able to use either a callback or a promise is also quite common, whereas I've never seen a library do as you propose.